test/kubernetes/kubernetes.go

```diff
@@ -119,3 +135,109 @@ func CleanupNamespaces(c kubernetes.Interface) error {
 
 	return nil
 }
+
+// NewIngress creates a new ingress
+func NewIngress(c kubernetes.Interface, ingress *networking.Ingress) error {
+	_, err := c.NetworkingV1().Ingresses(ingress.Namespace).Create(context.TODO(), ingress, metav1.CreateOptions{})
```

Comment:
Maybe add a check for an empty namespace? Just in case the BDD steps of future features were not all invoked in order and there's no namespace defined.

Reply:
fixed